dbms/src/Storages/DeltaMerge/workload/MainEntry.cpp

```diff
@@ -69,10 +68,6 @@ void initThreadPool()
         /*max_threads*/ default_num_threads,
         /*max_free_threads*/ default_num_threads / 2,
         /*queue_size*/ default_num_threads * 2);
-    IOThreadPool::initialize(
-        /*max_threads*/ default_num_threads,
-        /*max_free_threads*/ default_num_threads / 2,
-        /*queue_size*/ default_num_threads * 2);
```

Comment:
I think we still need to init at least `DataStoreS3Pool` and `S3FileCachePool` to make the dt_workload could run tests of writing/reading from S3 @JinheLin